cli/src/main/groovy/io/micronaut/cli/profile/DefaultFeature.groovy

```diff
@@ -133,5 +133,7 @@ class DefaultFeature implements Feature {
         if (maxJavaVersion != null) {
             return javaVersion <= maxJavaVersion
         }
+
+        true
```

Comment:
Is this change necessary?

Reply:
The only place I can think of it came in from a merge from master?

Reply:
when I compare this file to origin/master it says 100% match